README.md

```diff
@@ -1 +1,64 @@
-# hello-microverse
\ No newline at end of file
+
+# Hello Microverse
+
+>This is my first Project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [OPTIONAL] It is highly recommended to customize your readme file from the given template. Only choose all the areas applicable to your project at some point.